documentation/modules/security/con-securing-kafka-authorization.adoc

```diff
@@ -27,6 +27,7 @@ and are supported by all authorization mechanisms.
 
 To designate super users for a Kafka cluster, add a list of user principals to the `superUsers` property.
 If a user uses TLS client authentication, their username is the common name from their certificate subject prefixed with `CN=`.
+Otherwise, if not using the User Operator and using your own certificates for TLS client authentication, their username is their full certificate subject, which uses the form `CN=writeuser,OU=Unknown,O=Unknown,L=Unknown,ST=Unknown,C=Unknown`.
```

Comment:
Sorry, but I think this is confusing. It will look like this if you generate a certificate with all these fields set to `Unknown`. That IMHO doesn't make much sense. And it is not clear If certificate with subject `CN=writeuser,O=MyOrg` is `CN=writeuser,OU=Unknown,O=MyOrg,L=Unknown,ST=Unknown,C=Unknown` or not. I think you should use some better and more realistic example to make it more clear.

Reply:
I've updated the wording and added a couple more examples to show the case where a certificate may have less fields in the subject.
Let me know if it would be preferred to have an more realistic example in the example YAML (using fictional company, location names, etc)